python_modules/dagster/dagster/core/execution/context/invocation.py

```diff
@@ -231,8 +239,15 @@ def bind(
             alias=solid_def_or_invocation.given_alias
             if isinstance(solid_def_or_invocation, PendingNodeInvocation)
             else None,
+            user_events=self._user_events,
         )
 
+    def get_events(self) -> List[UserEvent]:
```

Comment:
Worth adding docstring. Also, I think that users might expect that yielded events would show up here. Is that difficult?

Reply:
good point. I don't think it's that difficult